src/clusterfuzz/_internal/issue_management/monorail/issue_tracker_manager.py

```diff
@@ -151,7 +153,10 @@ def _execute_with_retry(self, query):
   @retry.wrap(
       retries=FAIL_RETRIES,
       delay=FAIL_WAIT,
-      function='libs.issue_management.issue_tracker_manager._create_client')
+      # pylint: disable=line-too-long
+      function=
```

Comment:
Why not put this on one line?

Reply:
pylint will fail if keep one line. (even disabled line-too-long) It formatted by butler.py format.